Project1/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<body>
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link href="https://fonts.googleapis.com/css?family=Bitter" rel="stylesheet">
```

Comment:
Note that lines 10-13 are all Google Fonts requests and you only need one, even if you are using multiple fonts (ie, if you wanted to use Bitter, Bree Serif, EB Garamond, and PT Serif, you only need the request on line 11, all others could be deleted).

Since you are not using **any** of these fonts in your stylesheet, you should delete lines 10-13.

```suggestion
```